media-libs/jasper/jasper-2.0.33.recipe

```diff
@@ -0,0 +1,105 @@
+SUMMARY="Implementation of the codec specified in the JPEG-2000 Part-1 standard"
+DESCRIPTION="The JasPer Project is an open-source initiative to provide a \
+free software-based reference implementation of the codec specified in the \
+JPEG-2000 Part-1 standard. More details about this software can be found in \
+the JasPer Software Reference Manual."
+HOMEPAGE="http://www.ece.uvic.ca/~frodo/jasper/"
+COPYRIGHT="2001-2021 Michael D. Adams"
+LICENSE="JasPer v2"
+REVISION="1"
+SOURCE_URI="https://github.com/mdadams/jasper/archive/version-$portVersion.tar.gz"
+CHECKSUM_SHA256="38b8f74565ee9e7fec44657e69adb5c9b2a966ca5947ced5717cde18a7d2eca6"
+SOURCE_DIR="jasper-version-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="4.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	jasper$secondaryArchSuffix = $portVersion
+	lib:libjasper$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	jasper${secondaryArchSuffix}_devel = $portVersion
+	devel:libjasper$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	jasper$secondaryArchSuffix == $portVersion base
+	"
+
+if [ -z "$secondaryArchSuffix" ]; then
```

Comment:
If you are going to disable the gcc2 build, this `if` should be removed so that the tools are provided by the `_x86` package instead. Also, the `commandBinDir` pattern should be used then.

Reply:
I think that if we keep the older one around for gcc2 (for now), this is safe as it is atm, tools could still be provided by the base(old) version then?